README.md

```diff
@@ -1,5 +1,5 @@
 # [OpenVINO™ Toolkit](https://01.org/openvinotoolkit) - Open Model Zoo repository
-[![Stable release](https://img.shields.io/badge/version-2021.4-green.svg)](https://github.com/openvinotoolkit/open_model_zoo/releases/tag/2021.4)
+[![Stable release](https://img.shields.io/badge/version-2021.4-green.svg)](https://github.com/openvinotoolkit/open_model_zoo/releases/tag/2022.1)
```

Comment:
Still 2021.4 in `version-2021.4-green.svg`.

Also tagged releases has mismatched links: https://github.com/openvinotoolkit/open_model_zoo/blame/2021.4.2/README.md#L2
Release process should be improved.

Reply:
cc @Wovchena